src/data/python-quiz.ts

```diff
@@ -1,4 +1,24 @@
 const pythonQuiz = [
+  {
+    Question: "What is the output of the following. aTuple=(10, 20, 30, 40, 50, 60, 70, 80). print(aTuple[2:5])",
```

Comment:
The issue with using code examples for the quiz, is that it has to be all on one line. With your current code, this would result in an error because it would need to be broken up into two lines for it to run.

We would have to make this change

```suggestion
    Question: "What is the output of the following Python code: print((10, 20, 30, 40, 50, 60, 70, 80)[2:5])?",
```